scripts/core/superdesk-api.d.ts

```diff
@@ -3657,6 +3659,8 @@ declare module 'superdesk-api' {
          * (it will be rendered in different DOM locations depending if field is in header or content section)
          */
         miniToolbar?: JSX.Element;
+        sectionClassNames?: IAuthoringSectionClassNames;
```

Comment:
I was refactoring code and removed `sectionClassNames` completelly. You should not be adding it back.